### PR TITLE
release minio 0.1.1

### PR DESCRIPTION
**Pull request by Gupta-Amrit**
DC/OS minio service release 0.1.1
This release contains bug fix for statsd metrics which was failing every hour.

### CODE REVIEW

File: repo/packages/M/miniod/1/package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "packagingVersion": "4.0", 
+    "upgradesFrom": [
+        "{{upgrades-from}}"
+    ], 
+    "downgradesTo": [
+        "{{downgrades-to}}"
+    ], 
+    "minDcosReleaseVersion": "1.10", 
+    "name": "miniod", 
+    "version": "0.1.1",
```
Comment (realmbgl) [inline repo/packages/M/miniod/1/package.json]: hi @Gupta-Amrit, version is missing the basetech version part, see previous release for sample.

cc @ryadav88

File: repo/packages/M/miniod/1/package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+    "packagingVersion": "4.0", 
+    "upgradesFrom": [
```
Comment (realmbgl) [inline repo/packages/M/miniod/1/package.json]: hi @Gupta-Amrit change this to following
```
    "upgradesFrom": [], 
    "downgradesTo": [], 
```

File: repo/packages/M/miniod/1/package.json (hunk 1 above)
Comment (Gupta-Amrit) [reply repo/packages/M/miniod/1/package.json]: Done.

File: repo/packages/M/miniod/1/package.json (hunk 2 above)
Comment (Gupta-Amrit) [reply repo/packages/M/miniod/1/package.json]: Done.

**Pull request merged**